t/Makefile

```diff
@@ -26,8 +26,12 @@ test:
 # 	build the t.go binary
 	@go build .
 #	clean go testcache
-	go clean -testcache
-#	run the tests
-	@./t --skip systest/backup,systest/online-restore,systest/loader,tlstest
+	@go clean -testcache
+#	run t.go with specified arguments; otherwise run standard suite
+	@if [ -n "$(args)" ]; then \
+		./t $(args); \
+	else \
+		./t; \
+	fi
 #	clean up docker containers after test execution
-	./t -r
+	@./t -r
```

Comment:
Why do we use `@` ?

Reply:
Without the @ the makefile will print out the command and then execute it; with the @ the command is just executed.